Frameworks/UIKit/UIWebView.mm

```diff
@@ -251,6 +251,25 @@ - (NSString*)stringByEvaluatingJavaScriptFromString:(NSString*)string {
 }
 
 /**
+ @Status Caveat
+  @Notes This is a workaround. Original UIWebView does not have this method
+*/
+- (void)evaluateJavaScript:(NSString*)javaScriptString completionHandler:(void (^)(id, NSError*))completionHandler {
```

Comment:
stringByEvaluatingJavaScriptFromString isn't async though.

---
In reply to: [95190774](https://github.com/Microsoft/WinObjC/pull/1638#discussion_r95190774) [](ancestors = 95190774)